src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -670,11 +672,15 @@ public void InvalidMaxCPUCountSwitch4()
 
   </Target>
 
+  <Target Name=""Restore"">
```

Comment:
This is a little tricky, but our check for whether we're restoring is just to check:
targets.Length == 1 && string.Equals(targets[0], MSBuildConstants.RestoreTargetName, StringComparison.OrdinalIgnoreCase);

That means this is "restoring"